### PR TITLE
Fixes #954 : Addition of a Navbar for tracks

### LINKED ISSUE #954: Implement  a nav bar for sessions/tracks
Currently there is no navbar on sessions/tracks pages and it becomes difficult to navigate through the sessions without a navbar. We could take inspiration from eventyay navbar from the link given below, on the webapp they are datewise so there would be an option to select the date and then the navbar would display the corresponding tracks according to the selected tab on navbar.

screen shot 2017-01-07 at 12 08 41 pm

Eventyay sessions page : https://eventyay.com/e/18252ab6/sessions/

PS: If the issue seems valid, I would like to work on this.

### PR DESCRIPTION
**Pull request by CosmicCoder96**
Here is the live preview of, the navbar on a site which gets generated:

https://cosmiccoder96.github.io/open-event-webapp/tracks.html

screen shot 2017-01-11 at 1 45 21 am

@aayusharora It doesn't cover much of the area , in smaller devices, rather is scrollable.

### CODE REVIEW

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -1251,3 +1251,97 @@ a {
   right :0;
   top :0;
 }
+
+.c-tabs-nav {
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @CosmicCoder96 
#### Better class names
make class name understandble

File: src/backend/_scss/application.scss [hunk 2]
```diff
@@ -1251,3 +1251,97 @@ a {
...
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.c-tabs-nav__link {
+  flex: 1;
+  margin-right: 4px;
+  padding: 12px;
+  color: #fff;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: ## Required
Use SASS nesting properties instead of using same class names .c-tabs-nav__link

File: src/backend/_scss/application.scss [hunk 3]
```diff
@@ -1251,3 +1251,97 @@ a {
...
+  display: none;
+  background-color: #e7e7e7;
+}
+
+.c-tab.is-active {
+  display: block;
+
+}
+
+.c-tab__content {
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: ## Required
@CosmicCoder96  Dont use plain css under .scss files, please write proper SASS.

File: src/backend/_scss/application.scss (hunk 2 above)
Comment (aayusharora) [inline src/backend/_scss/application.scss]: ## Required
Use all colors from config.scss instead.

File: src/backend/templates/tracks.hbs [hunk 4]
```diff
@@ -24,11 +24,97 @@
   <link rel="stylesheet" href="./css/bootstrap.min.css">
   <link rel="stylesheet" href="./css/schedule.css">
 
+<script>(function() {
```
Comment (aayusharora) [inline src/backend/templates/tracks.hbs]: ## Required
- Make a seperate js file instead  of adding to track.hbs

File: src/backend/templates/tracks.hbs [hunk 5]
```diff
@@ -24,11 +24,97 @@
...
+          var link = tabNavigationLinks[i];
+          handleClick(link, i);
+        }
+      }
+    };
+
+    /**
+     * handleClick
+     *
+     * @description Handles click event listeners on each of the links in the
```
Comment (aayusharora) [inline src/backend/templates/tracks.hbs]: ## Required
Why is this needed ? Please remove extra comments @CosmicCoder96

**Comment by CosmicCoder96** [comment]
@aayusharora I have made the requested changes, Please review again.

**Comment by mariobehling** [comment]
Thanks. Please use the color codes that exist on the page. Please do not use the pink color for links.

**Comment by mariobehling** [comment]
Also, adjust the width to match the width of the following area.

**Comment by CosmicCoder96** [comment]
@aayusharora @mariobehling Thank you for the review. I have made the necessary changes.

**Comment by Princu7** [comment]
@CosmicCoder96 Great design and work. Just one small issue. On hovering over the dates, the text goes from white to black and since the background color is already black, the text becomes difficult to read. Please fix this. Thank you!!

**Comment by CosmicCoder96** [comment]
@Princu7 I have changed the colour. Please have a look.

**Comment by championswimmer** [comment]
Looks good now.
